### PR TITLE
LRAC-14114 Fix dropdown button style on AC user permission

### PR DESCRIPTION
**Pull request by fmarcio**
Jira issue https://liferay.atlassian.net/browse/LRAC-14114

### CODE REVIEW

**Comment by fmarcio** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/modals/BatchActionModal.tsx [hunk 1]
```diff
@@ -99,11 +99,9 @@ const BatchActionModal: React.FC<IBatchActionModalProps> = ({
 						trigger={
 							<ClayButton
 								className='button-root'
-								displayType='unstyled'
+								displayType='secondary'
 							>
-								<span className='text-truncate'>
-									{selectedKey}
-								</span>
+								<span>{selectedKey}</span>
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/modals/BatchActionModal.tsx:104]: Hey @fmarcio why did you remove `text-truncate` classname?

Comment (fmarcio) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/modals/BatchActionModal.tsx:104]: Hey @interaminense I removed it because this class was avoiding the icon in the button to be displayed correctly
Btn with `text-truncate` class:
change-permission-btn

**Comment by interaminense** [comment]
ci:forward:force
